.ci/azure-pipelines/azure-pipelines.yaml

```diff
@@ -12,7 +12,10 @@ trigger:
     - README.md
     - CHANGES.md
     - CONTRIBUTING.md
-    # add more exclude rules, e.g. .ci, .dev, .github
+    - .ci
```

Comment:
.ci and .github can mean change in the pipeline. We want to run pipelines in those cases

Reply:
Sorry for the mistake, I would fix it.